litellm/proxy/utils.py

```diff
@@ -2987,19 +2987,27 @@ def get_error_message_str(e: Exception) -> str:
     return error_message
 
 
-def _get_redoc_url() -> str:
+def _get_redoc_url() -> Optional[str]:
     """
-    Get the redoc URL from the environment variables.
+    Get the Redoc URL from the environment variables.
 
     - If REDOC_URL is set, return it.
+    - If NO_REDOC is True, return None.
     - Otherwise, default to "/redoc".
     """
-    return os.getenv("REDOC_URL", "/redoc")
+    redoc_url = os.getenv("REDOC_URL", None)
+    if redoc_url:
+        return redoc_url
+    
+    if os.getenv("NO_REDOC", "False") == "True":
```

Comment:
please us str_to_bool we use this in our codebase for reading env vars

Reply:
Done.

IMHO, current implementation of `str_to_bool` should be called `str_to_optional_bool`. A real `str_to_bool` should require a `default` boolean argument and never returns None. It's ugly we pass a boolean string default value to `os.getenv`.

I would prefer
```
str_to_bool(os.getenv("DISABLE_AIOHTTP_TRANSPORT"), default=False)
```
than
```
str_to_bool(os.getenv("DISABLE_AIOHTTP_TRANSPORT", "False")) is True
```